packages/suite/src/views/wallet/send/Options/BitcoinOptions/Locktime/Locktime.tsx

```diff
@@ -45,28 +59,46 @@ export const Locktime = ({ close }: LocktimeProps) => {
 
     const options = getDefaultValue('options', []);
     const broadcastEnabled = options.includes('broadcast');
-    const inputName = 'bitcoinLockTime';
-    const defaultInputValue = getDefaultValue(inputName) || '';
-    const error = errors[inputName];
 
-    const locktime = watch('bitcoinLockTime');
+    const blockCountInputName = 'bitcoinLockTimeBlockCount';
+    const blockCountDefaultInputValue = getDefaultValue(blockCountInputName) || '';
```

Comment:
Nit: Could you use `watch` instead? `getDefaultValue` is an old util which is redundant AFAIK.